ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml

```diff
@@ -0,0 +1,17 @@
+#ibexa:
+#    system:
+#        default:
+#            cdp:
+#                # Account number can be obtained from User Account settings in Ibexa CDP dashbaord
+#                account_number: 123456
+#                user_data_streaming:
+#                    # Use Stream ID generated during data import from "stream file" in Data Manager
+#                    stream_id: 00000000-00000000-00000000-00000000
+#                # You can configure multiple activations here. They have to be of type "Ibexa" in Ibexa DXP dashboard.
+#                activations:
+#                    -
+#                        # Client ID and Secret pair is used to authenticate against Webhook endpoint, make sure it's random and secure
+#                        client_id: 123456
+#                        client_secret: abcdef
```

Comment:
client_id and client_secret should be rather an env variable

Reply:
Fixed in [2113526](https://github.com/ibexa/recipes-dev/pull/23/commits/211352660d14c7d194035188dfc3b6fdd24bfe1b)